### ISSUE TITLE
Why enhancement shamans are broken!!

### ISSUE THREAD

**Issue by makechester** [WoWManiaUK/Blackwing-Lair#2324]
Hey stranger :)!
I'm Gorsky and I main enhancemant shamans.
I just hit lvl 85 now and it's time to tell you about the bugs of my spec that I found On this server:
1) [Feral Spirit]: They stun, okay good But.. Normally every hit they do, I get heals 2k-5k Per Hit, Here They don't heal, and about [Spirit Walk] it's not automatically works when I spawn them, So basically I gotta do [Ctrl 5] To activate it .. And the most triggered thing is they hit low, and when I say low, I mean it.. They literally hit with 155 and I'm lvl 85.
2) The AoE Dmg: So basically as an enhancement shaman when you wanna do AoE dmg
you spam [Flame Shock] on a mob then do [Lava Lash] So the [Flame Shock] will spread and then do [Fire Nova] Or do [Unleash Elements] before it for the ultra AoE Dmg.. Well, Here when you wanna do that and you spam your [Flame Shock] then [Lava Lash] it doesn't spread. So basically We got no AoE dmg here.
3) [Lava lash]+[Searing totem]: Basically you gotta wait for 5 stacks of [searing totem] then you spam [Lava lash] For extra dmg. Normally when you do that the 5 stacks will disappear.. Here if  it gets to 5 stacks it always stays like that even if you spam [Lava Lash].
Now prepare yourselves for number 4 !!
4) [Maelstorm weapon]: Basically it reduces the cast time and mana cost of specific spells with 20% each stack. So in the 5th it will be instant. HERE!! When you spam [Greater Healing Wave] (Which is basically the most powerfull Heal in the shaman) The stacks stay the same. So basically you heal yourself instantly without consuming any mana with your most powerfull Heal spell.

I think that's what I got So far, Thanks for reading :).

**Comment by sarahkay88** [comment]
You will need to make a separate report from each issue - using the template provided - i have attached the links to help you out:
https://wow.gamepedia.com/Feral_Spirit
https://wow.gamepedia.com/Spirit_Walk
https://wow.gamepedia.com/Flame_Shock
https://wow.gamepedia.com/Lava_Lash
https://wow.gamepedia.com/Searing_Totem
https://wow.gamepedia.com/Maelstrom_Weapon
https://wow.gamepedia.com/Greater_Healing_Wave

**Comment by makechester** [comment]
Well I didn't know about the links stroy at the start, plus I was talking on general about that class which makes it one issue and it's the enhancement shaman.. Yet when I get time I will try to edit it Else you can delete it.. idk. I just want you to get the point of how much that class is broken.
Ty :).

**Comment by Kitzunu** [comment]
Please follow the Template made to log issues.
You can also read CONTRIBUTION.MD for extra help.

Thank you for taking the time to log the issue, it will be looked into

**Comment by wow-mania** [comment]
issues like this will never be looked at.
log each spell separately with the actual issue described and proofs.
I appreciate the effort but this is not how we work.

**Comment by Dranah** [comment]
Well, i hate to say I told you, but I told you @makechester :D
Made separate report for each.
https://github.com/WoWManiaUK/Blackwing-Lair/issues/2389
https://github.com/WoWManiaUK/Blackwing-Lair/issues/2390
https://github.com/WoWManiaUK/Blackwing-Lair/issues/2388
Feral spirit was already reported here:https://github.com/WoWManiaUK/Blackwing-Lair/issues/2110

**Comment by TheSale** [comment]
issue regarding lava lash consuming flame shock is invalid and it is from MoP patch, same as searing totem where stacks should be consumed when lava lash is used. That is also in MoP patch.

**Comment by TheSale** [comment]
I think I figured out where is issue guys...
This made such a mess here.
You guys reported spells, and I think there is nothing wrong with spells and everything work as intended.
Issue is Talent!
Improved lava lash talent dont work
Spells were changed in MoP since entire talent system were changed, and some talents were implemented into spells directly.
http://prntscr.com/o2ks67
http://prntscr.com/o2kshr
here I tested lava lash, first is without searing flames and second is with 5 stacks
they are not consumed, but they also dont increase damage of lava lash 9k damage in both scenarios.

close issues regarding spells and just set talent issue
